Week1/SantiagoCollazosBarrera/Collections/Collections Challenges/README.md

```diff
@@ -0,0 +1,43 @@
+# Collection challenges
+
+It were using 10000000 elements inserted way random
+
+##### Lists
+
+Add an element can have the same time(Miliseconds), because de insert ocurrs at the end of the list.
```

Comment:
What about deleting the first elements (0, 1, 2) and the last elements (n-1, n-2, n-3) who is faster?